addons/mrp/models/mrp_workorder.py

```diff
@@ -145,6 +146,79 @@ def _read_group_workcenter_id(self, workcenters, domain, order):
     allowed_lots_domain = fields.One2many(comodel_name='stock.production.lot', compute="_compute_allowed_lots_domain")
     is_finished_lines_editable = fields.Boolean(compute='_compute_is_finished_lines_editable')
 
+    json_popover = fields.Char('Popover Data JSON', compute='_compute_json_popover')
+
+    def _get_conflicted_workorder_ids(self):
+        """Check if current workorder(s) conflict with other(s).
+
+        Conflict means having two workorders in the same time in the same workcenter.
+        Return a set of conflicted workorder ids (only for self).
+
+        :return: set of conflicted workorder ids (if in the set = conflict)
+        """
```

Comment:
also the return value/type is weird?
maybe returning a dict is more futur proof
self.ids = [1,2,3] -> the return value should be {1: [4,5], 2: [], 3: []} if [4,5] are in conflict